--- filemanager-actions-3.4/src/ui/fma-config-tool.desktop.in.orig	2017-07-29 21:57:47.213847013 +0200
+++ filemanager-actions-3.4/src/ui/fma-config-tool.desktop.in	2017-07-29 21:57:50.287180310 +0200
@@ -5,5 +5,5 @@
 _Comment=Configure items to be added to the File-Manager context menus
 Exec=fma-config-tool
 Icon=fma-config-tool
-Categories=System;FileTools;FileManager;
+Categories=GTK;Settings;System;FileTools;FileManager;
 StartupNotify=true
